packages/file_selector/file_selector_macos/example/lib/get_multiple_directories_page.dart

```diff
@@ -0,0 +1,88 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:file_selector_platform_interface/file_selector_platform_interface.dart';
+import 'package:flutter/material.dart';
+
+/// Screen that allows the user to select one or more directories using `getDirectoryPaths`,
+/// then displays the selected directories in a dialog.
+class GetMultipleDirectoriesPage extends StatelessWidget {
+  /// Default Constructor
+  const GetMultipleDirectoriesPage({Key? key}) : super(key: key);
+
+  Future<void> _getDirectoryPaths(BuildContext context) async {
+    const String confirmButtonText = 'Choose';
+    final List<String> directoriesPaths =
+        await FileSelectorPlatform.instance.getDirectoryPaths(
+      confirmButtonText: confirmButtonText,
+    );
+    if (directoriesPaths == null || directoriesPaths.isEmpty) {
+      // Operation was canceled by the user.
+      return;
+    }
+    String paths = '';
+    for (final String? path in directoriesPaths) {
+      paths += '${path!} \n';
+    }
```

Comment:
Per comment in another implementation's review, it seems like this should just be `directoryPaths.join('\n')`.